cosigned/base/kustomization.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: cosigned
+resources:
+- ../../cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/cosigned
+- ../../cluster-scope/base/rbac.authorization.k8s.io/clusterroles/cosigned
+- configmap.yaml
+- deployment.yaml
+- ../../cluster-scope/base/cosigned
+- publicsecret.yaml
+- rolebinding.yaml
+- role.yaml
+- secret.yaml
+- serviceaccount.yaml
+- service.yaml
```

```suggestion
- configmap.yaml
- deployment.yaml
- publicsecret.yaml
- rolebinding.yaml
- role.yaml
- secret.yaml
- serviceaccount.yaml
- service.yaml

```
The cluster resource don't need to be in here.